patch/driver-arista-mmcblk-not-working-on-AMD-platforms.patch

```diff
@@ -0,0 +1,32 @@
+amd/mmc: mmcblk not working on some AMD platforms
```

Comment:
Please properly format them with `git format-patch`.

Reply:
I will update the PR with the patch generated via `git format-patch`